makefile

```diff
@@ -202,7 +202,7 @@ $(buildDir)/.lintSetup:$(buildDir)/golangci-lint
 	@touch $@
 $(buildDir)/golangci-lint:
 	@curl $(curlRetryOpts) -o "$(buildDir)/install.sh" https://raw.githubusercontent.com/golangci/golangci-lint/$(goLintInstallerVersion)/install.sh
-	@echo "$(goLintInstallerChecksum) $(buildDir)/install.sh" | sha256sum --check
+	@echo "$(goLintInstallerChecksum) *$(buildDir)/install.sh" | shasum --check
```

Comment:
The preceding `*` is just part of the spec when checking and supplying binaries: https://linux.die.net/man/1/shasum